lib/alephant/renderer/views/html.rb

```diff
@@ -1,64 +1,57 @@
-require "alephant/renderer/views/base"
-require "mustache"
-require "i18n"
+require 'alephant/renderer/views/base'
+require 'alephant/renderer/engine/mustache'
+require 'alephant/renderer/i18n/locale_component_yaml'
 
 module Alephant
   module Renderer
     module Views
-      class Html < Mustache
+      class Html
         include ::Alephant::Renderer::Views::Base
 
+        class << self
+          # FIXME: remove this when we've gotten rid of all of the `self.template_path` directives
```

Comment:
The reason for using `self.template_path` was to point the models in the lib directory to the templates in the same directory. Or else they produce an error about the template not being found.

Not one for this PR, but we should look at fixing it sometime if possible, as it does seem a bit odd to have to specify a template path if in the lib directory.